app/views/PenalisedProfessionalBodyDetailView.scala.html

```diff
@@ -14,42 +14,56 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.charactercount.CharacterCount
+@import uk.gov.hmrc.govukfrontend.views.Implicits.RichCharacterCount
 @import controllers.routes._
 @import models.Mode
 @import config.FrontendAppConfig
+@import views.ViewUtils
+@import components.{Anchor, Heading, FormErrorSummary, Button, PageHeading}
 
 @this(
-        main_template: MainTemplate,
-        formHelper: FormWithCSRF,
-        appConfig: FrontendAppConfig
+layout: template.Layout,
+formHelper: FormWithCSRF,
+errorSummary: FormErrorSummary,
+govukCharacterCount: GovukCharacterCount,
+govukTextarea: GovukTextarea,
```

Comment:
Appreciate this is a minor thing but this is still not resolved. `govukTextArea` is injected here but not used in this view, can it please be removed